mod/consensus/pkg/cometbft/service/service.go

```diff
@@ -75,8 +76,6 @@ type Service[
 	initialHeight        int64
 	minRetainBlocks      uint64
 
-	// application's version string
-	version string
```

Comment:
does not seem necessary to make it an attribute of node since it's value won't change upon execution